mewarpx/mewarpx/emission.py

```diff
@@ -1397,7 +1397,7 @@ def plot_contours(self):
 
         contours = np.array(skimage.measure.find_contours(inside, 0.17))
 
-        inside = inside.T
+        inside = inside
```

Comment:
Why do this?

Reply:
The original plotting function plotted x on the x axis and z on the y axis, but I changed it to plot x on the y axis and z on the x axis to be more consistent with all our other plots.

Reply:
Pretty sure you can delete the line of code now that you've made the change though, @KZhu-ME :)

Reply:
Wow, I just realized...